less/styles.less

```diff
@@ -172,6 +181,15 @@ h2.highlight {
         color: #fff;
     }
 }
+
+.eclipsefdn-agenda-schedule {
+  @media (max-width: @screen-xs-max) {
+    table {
+      font-size: 12px;
```

Comment:
We should use 14px, not 12px. It still accomplishes what we need without making it hard to read. We shouldn't make the text smaller than the standard body text across our sites

Reply:
Thx, updated